pm_dftb_opt/pm_dftb_opt.xml

```diff
@@ -27,6 +27,11 @@
         <requirement type="package" version="0.2.1">pymuonsuite</requirement>
         <requirement type="package" version="3.0">zip</requirement>
     </requirements>
+    <required_files>
+        <include type="literal" path="get_out_folder.py"/>
+        <include type="literal" path="run.sh"/>
+        <include type="prefix" path="README-"/>
```

Comment:
Are the readme files really needed for the function of the tool?

Reply:
If we don't have the `README-` files present (I renamed them and then ran the tests with `planemo`), then the `command` will fail on this line:

https://github.com/muon-spectroscopy-computational-project/muon-galaxy-tools/blob/bd5632424b9e1ba94d7c47c91f0e7f08c5fda15d/pm_dftb_opt/pm_dftb_opt.xml#L39

`cp: cannot stat '/home/kez92691/.planemo/planemo_tmp_h4wynvqy/README-pbc': No such file or directory`

As to why we have this dependence, I believe it is (at least partially) due to the fact that the `3ob` and `pbc` datasets have a different license to the tool/dftb+ itself  which requires that we "indicate the Licensed Material is licensed under this Public License, and include the text of, or the URI or hyperlink to, this Public License" (this was also relevant to #96 ):

https://github.com/muon-spectroscopy-computational-project/muon-galaxy-tools/blob/bd5632424b9e1ba94d7c47c91f0e7f08c5fda15d/pm_dftb_opt/README-3ob#L8-L10

Reply:
Interesting... Never saw that. Thanks for the explanation